Dockerfile

```diff
@@ -1,35 +1,68 @@
-FROM node:14-alpine
+FROM ubuntu:22.04
+
+# Install basic dependencies
+RUN apt-get update && apt-get install -y \
+    curl \
+    python2 \
+    make \
+    g++ \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install nvm
+ENV NVM_DIR=/root/.nvm
+ENV NODE_VERSION=14.21.3
+
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash \
+    && . $NVM_DIR/nvm.sh \
+    && nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+
+# Add node and npm to path so the commands are available
+ENV NODE_PATH=$NVM_DIR/v$NODE_VERSION/lib/node_modules
+ENV PATH=$NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
+
 # First part, build the app
 WORKDIR /app
```

Comment:
I think mojaloop standard is /opt/app. And this dockerfile is deviating from the other repos. Please check